spec/javascripts/NewService/ServiceDiscoveryForm.spec.jsx

```diff
@@ -72,3 +74,15 @@ describe('fetchProjects', () => {
     expect(fetch).toHaveBeenCalled()
   })
 })
+
+
+describe('when Api as Product is enabled', () => {
```

Comment:
This is repetition, it's already asserted in `NewServiceForm`.

Reply:
If you move **SelectBackendApi** outside **ServiceManualForm**  and **ServiceDiscoveryForm** this is not needed